README.md

```diff
@@ -129,6 +127,16 @@ Find your maven `settings.xml` file. On Linux it should be at `~/.m2/settings.xm
 </repository>
 ```
 
+### Step 3.9: (If using Invoke) create a .env file
+A `.env` file will tell Invoke which server you are using. You can create multiple files, like
+`.env.foo` and `.env.bar`, and then Invoke will behave according to which one is symlinked
+(`ln -s`) to `.env`.
+
+To get started, copy `.env.sample` to `.env.something`, open it up, and change the values
+as appropriate for your development work. Then run `ln -s .env.something .env`.
+
+Once the initial `.env` file exists, you can change environment with `invoke setenv something`.
+
```

Comment:
Then you would add this below to a section on "Using PIH Invoke in your development setup", that has this step and documentation on the other tools